packages/blocks/src/page-block/utils/cursor.ts

```diff
@@ -65,7 +65,9 @@ export function getNativeSelectionMouseDragInfo(e: SelectionEvent) {
   const direction: DragDirection = getDragDirection(e);
   console.log(`direction: ${direction}`);
```

Comment:
We'd better remove this kind of log code in our codebase :)

Reply:
This PR is merged to the `feat/quick-format-bar`, I will clear the log before merging to master.